core/src/main/java/bio/terra/pearl/core/service/workflow/EnrollmentService.java

```diff
@@ -136,20 +141,32 @@ public HubResponse<Enrollee> enroll(String portalShortcode, EnvironmentName envN
      * the "enrollee"
      */
     @Transactional
-    public HubResponse<Enrollee> enrollAsProxy(String portalShortcode, EnvironmentName envName, String studyShortcode, ParticipantUser user, PortalParticipantUser ppUser, UUID preEnrollResponseId) {
-        HubResponse<Enrollee> proxyResponse = enroll(portalShortcode, envName, studyShortcode, user, ppUser, preEnrollResponseId, false);
-        HubResponse<Enrollee> governedResponse = enrollGovernedUser(portalShortcode, envName, studyShortcode, proxyResponse.getEnrollee(), user, ppUser, preEnrollResponseId);
-        governedResponse.setEnrollee(proxyResponse.getEnrollee());
+    public HubResponse<Enrollee> enrollAsProxy(EnvironmentName envName, String studyShortcode, ParticipantUser proxyUser,
```

Comment:
please update the comment of this method to indicate that it doesn't create an enrollee for the given user if one already exists